src/features/Auth/Login/Login.tsx

```diff
@@ -1,10 +1,37 @@
-import { Component } from 'react';
+import { Component, useEffect, useState } from 'react';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
-import { Link } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
 import Header from '../../../Shared/Header';
 
+function LoginF ( ) {
+  const [email,setEmail]=useState(""); 
+  const [password,setPassword]=useState("");
+  const history =useHistory(); 
+  useEffect (() => {
+    if (localStorage.getItem('user-info')) {
+      history.push("/add")
+    }
+  }, [])
+
+  async function LoginF(){
+    console.warn(email,password)
+    let item={email,password};
+    let result= await fetch("http://localhost:3000/login",{
+      method:'POST',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/xLibMS/LibMS-frontend/pullRequest?prid=7273853)
```suggestion
      method:"POST",
```